internal/sso/update/update.go

```diff
@@ -0,0 +1,45 @@
+package update
+
+import (
+	"context"
+	"errors"
+	"os"
+
+	"github.com/supabase/cli/internal/sso/internal/render"
+	"github.com/supabase/cli/internal/utils"
+	"github.com/supabase/cli/pkg/api"
+)
+
+func Run(ctx context.Context, ref, providerId string, params api.UpdateProviderByIdJSONRequestBody, addDomains, removeDomains []string, format string) error {
```

Comment:
Let's validate the project ref here as a sanity check. for eg. https://github.com/supabase/cli/blob/main/internal/ssl_enforcement/get/get.go#L13